src/20240102_AaveV2Polygon_ReserveFactorUpdates/ReserveFactorUpdates.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Polygon V2 Reserve Factor Updates"
+author: "karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-reserve-factor-updates-polygon-aave-v2/13937"
+---
+
+## Simple Summary
+
+This AIP is a continuation of [AIP 284](https://app.aave.com/governance/proposal/284) and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
```

Comment:
Links for old proposals are broken at the moment on Avara's UI, better to avoid linking to previous